docs/specs/xref.yml

```diff
@@ -436,7 +436,7 @@ inputs:
 outputs:
   docs/a.json:
   docs/b.json: |
-    { "content": "<p>Link to <a href=\"xref:c#bookmark\">title</a></p>" }
+    { "content": "<p>Link to <a href=\"\">title</a></p>" }
```

Comment:
is this a behavior change?

Reply:
it is, does it currently impact our comparison with v2?

Reply:
I think it should be as it is written while uid not resolved

Reply:
will this affect v2 as well?

Reply:
nop, it'll not affect v2.

Reply:
then we are having a behavior change compared to v2